src/index.ts

```diff
@@ -16,6 +16,7 @@ export { Button, ButtonAnchor, ButtonDiv } from './components/Button'
 export { Tag } from './components/Tag'
 export { Base } from './components/Base'
 export { Icon } from './components/Icon'
+export { Table } from './components/Table'
```

Comment:
[IMO] Does the person who uses smarthr-ui use the following methods?

```jsx
import { Table } from 'smarthr-ui'

const Component = () => (
  <Table.Table>
    <Table.Head>
      <Table.Row>
        <Table.Cell>hoge</Table.Cell>
      </Table.Row>
    </Table.Head>
    <Table.Body>
      <Table.Row>
        <Table.Cell>hoge</Table.Cell>
      </Table.Row>
    </Table.Body>
  </Table.Table>
)
```

or

```jsx
import { Table } from 'smarthr-ui'

const { Table, Head, Row, Cell } = Table

const Component = () => (
  <Table>
    <Head>
      <Row>
        <Cell>hoge</Cell>
      </Row>
    </Head>
    <Body>
      <Row>
        <Cell>hoge</Cell>
      </Row>
    </Body>
  </Table>
)
```

Isn't it anyway redundant?
Also, it felt unnatural that there was a Table in the Table

Reply:
What about the following interface?

```jsx
import { Table, Head, Row, Cell } from 'smarthr-ui'
```

or

```jsx
import { Table, TableHead, TableRow, TableCell } from 'smarthr-ui'
```